src/DetectChanges/BCBreak/PropertyBased/PropertyDocumentedTypeChanged.php

```diff
@@ -34,8 +35,13 @@ public function __invoke(ReflectionProperty $fromProperty, ReflectionProperty $t
             return Changes::empty();
         }
 
-        $fromTypes = array_unique($fromProperty->getDocBlockTypeStrings());
-        $toTypes   = array_unique($toProperty->getDocBlockTypeStrings());
+        try {
+            $fromTypes = array_unique($fromProperty->getDocBlockTypeStrings());
+            $toTypes   = array_unique($toProperty->getDocBlockTypeStrings());
+        } catch (InvalidArgumentException $failedToParseDocblock) {
+            // @TODO #134 improve docblock parsing upstream to remove this generic try-catch
```

Comment:
Not ideal to just hide it, but indeed we can't do much until we fix upstream, so this is kinda OK.